lib/chef/knife/bootstrap.rb

```diff
@@ -498,6 +507,35 @@ def render_template
         Erubis::Eruby.new(template).evaluate(bootstrap_context)
       end
 
+      # Check deprecated flags are used; map them to their new keys,
+      # and print a warning. Will not map a value to a new key if the
+      # CLI flag for that new key has also been specified.
+      # If both old and new flags are specified, this will warn
+      # and take the new flag value.
+      # This can be moved up to the base knife class if it's agreeable.
+      def warn_and_map_deprecated_flags
```

Comment:
Is this actually used, or is it replaced by verify_deprecated_flags!

Reply:
This might be bad rebase pasta introduced by me...

Reply:
I think this and the added `require`s at the top of here are an issue. All this PR should be adding is the license requires and code

Reply:
I *think* you are agreeing with me that they should be removed; is that correct?